geode-core/src/main/java/org/apache/geode/internal/cache/TXRegionLockRequestImpl.java

```diff
@@ -74,31 +76,36 @@ public boolean isEmpty() {
   }
 
   @Override
-  public void addEntryKeys(Set<Object> s) {
-    if (s == null || s.isEmpty()) {
+  public void addEntryKeys(Map<Object, Boolean> map) {
+    if (map == null || map.isEmpty()) {
       return;
     }
     if (this.entryKeys == null) {
-      // Create new temporary HashSet. Fix for defect # 44472.
-      final HashSet<Object> tmp = new HashSet<Object>(s.size());
-      tmp.addAll(s);
+      // Create new temporary HashMap. Fix for defect # 44472.
+      final HashMap<Object, Boolean> tmp = new HashMap<Object, Boolean>(map.size());
+      tmp.putAll(map);
       this.entryKeys = tmp;
 
     } else {
       // Need to make a copy so we can do a union
-      final HashSet<Object> tmp = new HashSet<Object>(this.entryKeys.size() + s.size());
-      tmp.addAll(s);
-      tmp.addAll(this.entryKeys);
+      final HashMap<Object, Boolean> tmp =
+          new HashMap<Object, Boolean>(this.entryKeys.size() + map.size());
+      tmp.putAll(this.entryKeys);
       this.entryKeys = tmp;
+      for (Map.Entry<Object, Boolean> entry : map.entrySet()) {
+        addEntryKey(entry.getKey(), entry.getValue());
+      }
     }
   }
 
   @Override
-  public void addEntryKey(Object key) {
+  public void addEntryKey(Object key, Boolean isevent) {
```

Comment:
Please use camel case style name -- like isEvent

Reply:
Thanks for comments.